src/FSharp.Data.GraphQL.Server/Execution.fs

```diff
@@ -597,16 +599,18 @@ let private executeSubscription (resultSet: (string * ExecutionInfo) []) (ctx: E
     let subdef = info.Definition :?> SubscriptionFieldDef
     let args = getArgumentValues subdef.Args info.Ast.Arguments ctx.Variables
     let returnType = subdef.OutputTypeDef
+    let fieldPath = [ info.Identifier :> obj ]
     let fieldCtx =
         { ExecutionInfo = info
           Context = ctx
           ReturnType = returnType
           ParentType = objdef
           Schema = ctx.Schema
           Args = args
-          Variables = ctx.Variables }
+          Variables = ctx.Variables
+          Path = fieldPath }
```

Comment:
I think fieldPath elements are in reverse order, so likely need to reverse before assigning.

Reply:
Right! Fixed now.